packages/cli/src/commands/build.ts

```diff
@@ -1,13 +1,15 @@
 import {Command} from '@oclif/command'
 import {build} from '@blitzjs/server'
+import {ServerConfig} from '@blitzjs/server/src/config'
 
 export default class Build extends Command {
   static description = 'Create a production build'
   static aliases = ['b']
 
   async run() {
-    const config = {
+    const config: ServerConfig = {
       rootFolder: process.cwd(),
+      serverless: true,
```

Comment:
I am just wondering is it the right approach to assume it is serverless just because it is a build? What if we deploy elsewhere other than zeit? Perhaps this should be an optional or passed in via blitz.config.js or a cli flag?

Reply:
Probably best to keep it in the blitz.config.js

Reply:
Ah, yes I forgot about `target: 'serverless'` in the config! Let's use that.

Need to check if `target.includes('serverless')` because it can also be something like `"serverless-trace-experimental"`